packages/ember-runtime/lib/system/core_object.js

```diff
@@ -191,6 +191,10 @@ function makeCtor() {
       this.init.apply(this, args);
     }
 
+    if (this.__postInitInitialization) {
```

Comment:
Yes, I will:

* implement as a symbol (we can always tweak later)
* add to base object to avoid property lookup (then remove `if`)